src/LettersOfAlphabet.java

```diff
@@ -0,0 +1,74 @@
+import org.junit.Test;
+import org.junit.runner.JUnitCore;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class LettersOfAlphabet {
+    public static void main(String[] args) {
+        JUnitCore.main("LettersOfAlphabet");
+    }
+
+    public List<Character> findMissingLetters(String word) {
+        List<Character> missingLetters = new ArrayList<>();
+        List<Character> alphabet = Arrays.asList(
+                'a','b','c','d','e','f','g','h','i','j',
+                'k','l','m','n','o','p','q','r','s','t','u','v','w','x','y','z'
+        );
+
+        //convert the string to a list of characters
+        String lowerCaseWord = word.toLowerCase();
+        Character[] letters = new Character[word.length()];
+
+        for(int i = 0; i < word.length(); i++){
+            letters[i] = lowerCaseWord.charAt(i);
+        }
+
+        List<Character> wordAsList = Arrays.asList(letters);
+
+        //Loop through the alphabet and add any missing letters
+        for(int i = 0; i < alphabet.size(); i++) {
+            if(!(wordAsList.contains(alphabet.get(i)))){
```

Comment:
you can probably ditch 1 pair of brackets
```suggestion
            if(!wordAsList.contains(alphabet.get(i))){
```